README.md

```diff
@@ -1,5 +1,9 @@
 # redux-dynamic-reducer
 
+## Deprecated
+
+Our new redux library, [redux-dynostore](https://github.com/ioof-holdings/redux-dynostore) has all the features of this one and more, allowing for a lot more dynamic store features. If you experience any difficulty switching over to [redux-dynostore](https://github.com/ioof-holdings/redux-dynostore) then please let us know by raising an issue over there. This library will be subject to major bug fixes and security fixes only.
```

Comment:
> all the features of this one and more

Are we sure?

Reply:
Nope, but if its missing some I want other people to help us find them... all the major features are (to my knowledge)